diff -u src1/src/obi_clean.c src2/src/obi_clean.c
--- src1/src/obi_clean.c	2019-09-27 14:52:05.000000000 +0200
+++ src2/src/obi_clean.c	2020-01-23 17:54:55.247616189 +0100
@@ -409,7 +409,7 @@
 			stop = true;
 		}
 
-		#pragma omp parallel default(none) \
+		#pragma omp parallel \
 					 	 	 shared(thread_count, seq_count, blob_array, complete_sample_count_array, alignment_result_array, \
 					 	 			 stop, blob1, i, obi_errno, keep_running, stderr, max_ratio, iseq_column, i_view, \
 									 similarity_mode, reference, normalize, threshold, ktable, status_column, o_view, sample_count)
